src/v/config/node_config.cc

```diff
@@ -180,6 +180,15 @@ node_config::node_config() noexcept
       "injection",
       {.visibility = visibility::tunable},
       std::nullopt)
+  , verbose_logging_timeout_sec_max(
+      *this,
+      "verbose_logging_timeout_sec_max",
+      "Maximum duration in seconds for verbose (i.e. TRACE or DEBUG) logging. "
+      "Values configured above this will be clamped. If null (the default) "
+      "there is no limit. Can be overridded in the Admin API on a per-request "
+      "basis.",
+      {.visibility = visibility::tunable},
+      std::nullopt)
```

Comment:
question: Is it possible to set a `min` here of 1?

Reply:
Great idea

Reply:
done